### PR TITLE
completed html-web-page challenge

### PR DESCRIPTION
**Pull request by kvnliang4**
image

### CODE REVIEW

File: html-web-page/index.html [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+        <td>Monthly Pricing</td>
+        <td>$30</td>
+        <td>$50</td>
+        <td>$60</td>
+      </tr>
+    </tbody>
+  </table>
+
+  <p>Like what you see?Become a part of the JOTMC family today!</p>
+
```
Comment (BrettAlbright) [inline html-web-page/index.html:82]: make sure that you include the `form` tags for the form, you don't want an input that is not part of a form

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
small update needed!

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Comment by kvnliang4** [comment]
image

I updated the code with a placeholder attribute instead of value to make the text faded.

**Pull request merged**